build/AzurePipelinesTemplates/WindowsAppSDK-RunTests-Steps.yml

```diff
@@ -0,0 +1,141 @@
+            }
+
+  - task: powerShell@2
+    displayName: 'Run dotnet installer'
+    inputs:
+      targetType: 'inline'
+      script: |
+            $(Build.SourcesDirectory)\packages\dotnet-windowsdesktop-runtime-installer.exe /quiet /install /norestart
+
+  - task: powerShell@2
+    displayName: 'Install VCLibs.Desktop'
+    inputs:
+      targetType: 'inline'
+      script: |
+            $package = "$(Build.SourcesDirectory)\packages\Microsoft.VCLibs.${{ parameters.buildPlatform }}.14.00.Desktop.appx"
+            Add-AppxPackage $package -ErrorAction SilentlyContinue
+
+  - task: powerShell@2
+    displayName: 'Install vc_redist'
+    inputs:
+      targetType: 'inline'
+      script: |
+            $(Build.SourcesDirectory)\packages\vc_redist.${{ parameters.buildPlatform }}.exe /quiet /install /norestart
+
+  - task: powershell@2
+    displayName: 'Run TE.Service'
+    inputs:
+      targetType: filePath
+      filePath: DevCheck.ps1
+      arguments: -NoInteractive -Offline -Verbose -CheckTAEFService
+      workingDirectory: '$(Build.SourcesDirectory)'
+
+  - task: VisualStudioTestPlatformInstaller@1
+    inputs:
+      versionSelector: latestStable
+
+  - task: PowerShell@2
+    displayName: Add test locale to User Language List
+    inputs:
+      targetType: 'inline'
+      script: |
+            $langToAdd = "${{ parameters.testLocale }}"
+            
+            Write-Host "Adding $langToAdd to user language list"
+            $langList = Get-WinUserLanguageList
+            $langList.Insert(0, $langToAdd)
+            Set-WinUserLanguageList -LanguageList $langList -Force
+
+            Write-Host "Get-WinUserLanguageList:" 
+            Get-WinUserLanguageList
+
+  - task: PowerShell@2
+    displayName: Display OS build/version/language info
+    inputs:
+      targetType: 'inline'
+      script: |
+            Get-Item -Path 'HKLM:\Software\Microsoft\Windows NT\CurrentVersion'
+            Write-Host (reg query "HKLM\SYSTEM\CurrentControlSet\Control\MUI" /s)
+            Write-Host (reg query "HKLM\SYSTEM\CurrentControlSet\Control\Nls" /s)
+            Write-Host (reg query "HKCU\Control Panel\International" /s)
+
+            Write-Host "Get-WmiObject -Class Win32_OperatingSystem" 
+            $OSInfo = Get-WmiObject -Class Win32_OperatingSystem
+            $languagePacks = $OSInfo.MUILanguages
+            $languagePacks
+
+            Write-Host "dism /online /Get-Intl" 
+            dism /online /Get-Intl
+
+            Write-Host "Get-WinUserLanguageList" 
+            Get-WinUserLanguageList
+
+            Write-Host "Get-WinUILanguageOverride"
+            Get-WinUILanguageOverride
+
+            Write-Host "Get-WinSystemLocale"
+            Get-WinSystemLocale
+  
+  - task: PowerShell@2
+    displayName: 'Run TAEF Tests'
+    inputs:
+      filePath: 'TestAll.ps1'
+      arguments: >
+        -OutputFolder "$(Build.SourcesDirectory)\BuildOutput\BuildOutput"
```

Comment:
Are we stuck with "...\BuildOutput\BuildOutput", or can we use a more meaning name like "...\BuildOutput\TestResults" or something.

Reply:
We are currently stuck with BuildOutput/BuildOutput until we can split the BuildOutput into dev/test folders.